spec/views/ft_forms_mailer/send_confirmation_text_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'ft_forms_mailer/send_confirmation.text.erb' do
+  before :each do
+    @form_data = Hash[question1: 'answer1', email: 'my_email']
```

Comment:
For more background, the [`Hash[]` method](http://ruby-doc.org/core-2.2.2/Hash.html#method-c-5B-5D) is a lot like the `Array()` method.  In the form you're using here, it takes something that *can* be converted into a `Hash` (including actual `Hash`es) and makes it one.  It's intended for situations where you don't know for certain what the input will be.